src/Components/PaperEditor/ProgrammeScriptContainer/index.js

```diff
@@ -75,6 +75,11 @@ const ProgrammeScriptContainer = (props) => {
     }
   };
 
+  const hadleSaveButton = () => {
```

Comment:
```suggestion
  const handleSaveButton = () => {
```
Small spelling tweak